conferences/2025-agu.md

```diff
@@ -34,6 +34,103 @@ We're excited to hear what you have to say at our session!
 **See you in New Orleans!** 🚀
 
 
-## Invited speakers
+## Invited speakers: _TBD_
 
-_TBD_
+::::::{grid} 1 1 1 2
+
+:::{card}
+:header: **Invited Speaker - TBD!**
+
+_Check back soon!_
+:::
+
+:::{card}
+:header: **Invited Speaker - TBD!**
+
+_Check back soon!_
+:::
+
+::::::
+
+
+## Organizers
+
+::::::{grid} 1 1 1 2
+
+:::{card}
+:header: **Primary Convener - Qiusheng Wu**
+
+![Headshot of Dr. Qiusheng Wu](https://github.com/user-attachments/assets/f65928b0-1771-4bff-8331-a9168c16c9d8)
+
+**Dr. Qiusheng Wu** is an Associate Professor and the Director of Graduate Studies in the
+Department of Geography & Sustainability at the University of Tennessee, Knoxville. He
+also serves as an Amazon Scholar. Dr. Wu’s research focuses on geospatial data science
+and open-source software development, with an emphasis on leveraging big geospatial data
+and cloud computing to study environmental change, particularly surface water and
+wetland inundation dynamics. He is the creator of several widely used open-source Python
+packages, including geemap, leafmap, segment-geospatial, and geoai, which support
+advanced geospatial analysis and interactive visualization.
+
+His open-source work is available at [\@opengeos](https://github.com/opengeos).
+:::
+
+
+:::{card}
+:header: **Convener - Max Jones**
+
+_Check back soon!_
+:::
+
+
+:::{card}
+:header: **Convener - Chelle Gentemann**
```

Comment:
@cgentemann I'm merging this PR so we can share out what we have so far. Please reply to this comment with a mini-bio and headshot and I'll open a new PR to integrate :)

Reply:
Thanks, @cgentemann !!